src/components/common/FilterGroup/index.tsx

```diff
@@ -0,0 +1,199 @@
+import React, { useState } from 'react';
+import { FieldValuesPayload, SearchNodePayload } from 'src/services/SearchService';
+import { Box, Theme, Typography, useTheme } from '@mui/material';
+import strings from 'src/strings';
+import MultipleSelection from './filters/FilterMultipleSelection';
+import SingleSelection from './filters/FilterSingleSelection';
+import Search from './filters/FilterSearch';
+import DateRange from './filters/FilterDateRange';
+import FilterNumberRange from './filters/FilterNumberRange';
+import FilterCountWeight from './filters/FilterCountWeight';
+import { Option } from '@terraware/web-components/components/table/types';
+import { makeStyles } from '@mui/styles';
+import { Button } from '@terraware/web-components';
+import useDeviceInfo from 'src/utils/useDeviceInfo';
+
+interface StyleProps {
+  isMobile?: boolean;
+  isDesktop?: boolean;
+}
+
+const useStyles = makeStyles((theme: Theme) => ({
+  divider: {
+    maxWidth: '90%',
+    border: 'none',
+    borderTop: `1px solid ${theme.palette.TwClrBrdrTertiary}`,
+  },
+  button: {
+    width: (props: StyleProps) => (props.isMobile ? '100%' : 'auto'),
+  },
+}));
+
+export type FilterField = {
+  name: string;
+  label: string;
+  type:
+    | 'multiple_selection'
+    | 'single_selection'
+    | 'search'
+    | 'date_range'
+    | 'number_range'
+    | 'count_weight'
+    | 'hidden';
```

Comment:
curious.. how does hidden work?

Reply:
Hmm, i just copy-pasted this from the database code, the case was simply not handled there, which is the same as what I've done.